net/http_test.ts

```diff
@@ -225,3 +225,51 @@ test(async function requestBodyStreamWithTransferEncoding() {
     }
   }
 });
+
+test(async function requestRespondWithoutReadingBodyStream() {
+  {
+    const req = new ServerRequest();
+    req.headers = new Headers();
+    req.headers.set("content-length", "5");
+    const rbuf = new Buffer(enc.encode("Hello"));
+    req.r = new BufReader(rbuf);
+    const wbuf = new Buffer(new Uint8Array(1024));
+    req.w = new BufWriter(wbuf);
+    await req.respond({
+      body: enc.encode("World")
+    });
+    // Body should have been secretly consumed by here
+    const body = dec.decode(await req.body());
+    assertEqual(body, "");
```

Comment:
Should we allow to consume body multiple times?

Reply:
Do you mean that you expect `req.body()` to return the content even after respond, or do you mean that invoking multiple times of `req.body()` would all yield empty string here?

Reply:
Sure. Updated.